C/tests/c4PerfTest.cc

```diff
@@ -26,7 +26,6 @@
 #include "Benchmark.hh"
 #include "FilePath.hh"
 #include "SecureRandomize.hh"
-#include "repo_version.h"
```

Comment:
Whuuuut?  Wouldn't that be a compiler error?

Reply:
Leaving it in was actually a compiler error, and I didn't know how it became so.  The include is not needed though so the simplest thing was to remove it.